genesyscloud/recording_media_retention_policy/resource_genesyscloud_recording_media_retention_policy.go

```diff
@@ -140,9 +158,14 @@ func updateMediaRetentionPolicy(ctx context.Context, d *schema.ResourceData, met
 	order := d.Get("order").(int)
 	description := d.Get("description").(string)
 	enabled := d.Get("enabled").(bool)
-	mediaPolicies := buildMediaPolicies(d, pp, ctx)
+	err, mediaPolicies := buildMediaPolicies(d, pp, ctx)
+
```

Comment:
looks like we're not checking this error

Reply:
Done